cypress/e2e/group3/metrics.ts

```diff
@@ -6,7 +6,6 @@ describe('Dockstore Metrics', () => {
   setTokenUserViewPort();
   it('Should see no metrics banner', () => {
     cy.visit('/workflows/github.com/A/l:master');
-    cy.get('.mat-tab-header-pagination-after').click();
```

Comment:
Pagination control no longer present, so don't click it. Applies to next to changes as well.